updates/1/updates.sql

```diff
@@ -1 +1,24 @@
-FILL ME IN
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xaac41EC512808d64625576EDdd580e7Ea40ef8B2", 18, NULL);
+INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xaac41EC512808d64625576EDdd580e7Ea40ef8B2", "C", "Gameswap", "GSWAP", 11155251, NULL, "gameswap", "gswap", "0xaac41EC512808d64625576EDdd580e7Ea40ef8B2");
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x88ACDd2a6425c3FaAE4Bc9650Fd7E27e0Bebb7aB", 18, NULL);
+INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x88ACDd2a6425c3FaAE4Bc9650Fd7E27e0Bebb7aB", "C", "Alchemist", "MIST", 11804891, NULL, "alchemist", "mist", "0x88ACDd2a6425c3FaAE4Bc9650Fd7E27e0Bebb7aB");
+UPDATE assets SET swapped_for="_ceth_0x88dF592F8eb5D7Bd38bFeF7dEb0fBc02cf3778a0" WHERE identifier="_ceth_0x0Ba45A8b5d5575935B8158a88C631E9F9C95a2e5";
```

Comment:
This needs an addition line.

Reply:
Isn't next line the needed addition? https://github.com/rotki/assets/pull/1/files/1338ab5694af1296d44659852cceadca7a4a892c#diff-ae49fcf47129e76696bdb39ff51f5e3b14b6b8017d0272a6c8e9b3a797d3db65R6

Reply:
The next line seems to only add to assets table and not also to the ethereum tokens one. We need the entire addition as if the token never existed. This is used in exactly the case that it's not in the user's DB.

Also it seems to edit a token whose identifier is `_ceth_0x88ACDd2a6425c3FaAE4Bc9650Fd7E27e0Bebb7aB` but the asset you are changing has an identifier `_ceth_0x0Ba45A8b5d5575935B8158a88C631E9F9C95a2e5`

Reply:
This is still wrong. As i said above you are modifying token `_ceth_0x0Ba45A8b5d5575935B8158a88C631E9F9C95a2e5`. So the simple insertion line should be for that token. Not for another one..

You seem to misunderstand how atoken swap needs to be added. You need two changes, so 4 lines.

```
1. UPDATE swapped_for in the original token
2. original token full insertion line
3. Insert new token
4. *  (since is the same as 3)
```